src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/ClaimantResponseConfirmsToProceedRespondentNotificationHandler.java

```diff
@@ -81,6 +84,11 @@ private CallbackResponse notifyRespondentSolicitorForClaimantConfirmsToProceed(C
             recipient = caseData.getRespondentSolicitor2EmailAddress();
         }
 
+        if (SPEC_CLAIM.equals(caseData.getCaseAccessCategory())
+            && caseData.isLRvLipOneVOne(multiPartyScenario)) {
```

Comment:
hmmm we have a scattered responsibility here. perhaps the knowledge should stay in the enum and instead have a method there? we already have isOnvOne there etc